src/providers/twitch/TwitchChannel.cpp

```diff
@@ -735,39 +735,8 @@ void TwitchChannel::parseLiveStatus(bool live, const HelixStream &stream)
     {
         auto status = this->streamStatus_.access();
         status->viewerCount = stream.viewerCount;
-        if (status->gameId != stream.gameId)
-        {
-            status->gameId = stream.gameId;
-
-            if (!stream.gameId.isEmpty())
-            {
-                // Resolve game ID to game name
-                getHelix()->getGameById(
```

Comment:
I believe this is the only use of `getGameById` - should the method be removed from the Helix file? (it simply just calls `fetchGames` anyways)